### PR TITLE
Fix the code warnings when building

### LINKED ISSUE #34: Fix the code warnings when building
Can we fix the following warnings?

```
  InternalEnforcer.cs(32, 48): [CS0168] The variable 'e' is declared but never used
  InternalEnforcer.cs(71, 48): [CS0168] The variable 'e' is declared but never used
  InternalEnforcer.cs(109, 48): [CS0168] The variable 'e' is declared but never used
  Config.cs(99, 36): [CS0168] The variable 'e' is declared but never used
  DefaultFileAdapter.cs(29, 32): [CS0168] The variable 'e' is declared but never used
  InternalEnforcer.cs(32, 48): [CS0168] The variable 'e' is declared but never used
  InternalEnforcer.cs(71, 48): [CS0168] The variable 'e' is declared but never used
  DefaultFileAdapter.cs(29, 32): [CS0168] The variable 'e' is declared but never used
  InternalEnforcer.cs(109, 48): [CS0168] The variable 'e' is declared but never used
  Config.cs(99, 36): [CS0168] The variable 'e' is declared but never used
  NuGet.Build.Tasks.Pack.targets(198, 5): [NU5125] The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
  NuGet.Build.Tasks.Pack.targets(198, 5): [NU5048] The 'PackageIconUrl'/'iconUrl' element is deprecated. Consider using the 'PackageIcon'/'icon' element instead. Learn more at https://aka.ms/deprecateIconUrl
```

image

### PR DESCRIPTION
**Pull request by Sagilio**
close #34

### CODE REVIEW

File: NetCasbin/NetCasbin.csproj [hunk 1]
```diff
@@ -23,8 +18,20 @@
     <FileVersion>1.2.6.0</FileVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageId>Casbin.NET</PackageId>
+    <PackageIcon>casbin.png</PackageIcon>
```
Comment (hsluoyz) [inline NetCasbin/NetCasbin.csproj:23]: Where is `casbin.png`?

**Comment by hsluoyz** [comment]
@huazhikui , Where is `casbin.png`? Is there anything I was missing here?

File: NetCasbin/NetCasbin.csproj (hunk 1 above)
Comment (Sagilio) [reply NetCasbin/NetCasbin.csproj:23]: It is here https://github.com/casbin/Casbin.NET/blob/master/NetCasbin/casbin.png.

Comment (hsluoyz) [reply NetCasbin/NetCasbin.csproj:23]: OK

**Pull request merged**